### PR TITLE
Added Stale issues and PRs GitHub action

### PR DESCRIPTION
**Pull request by DennisSSDev**
*Description of changes:*
There are some issues and PRs that have been not active on the repo for a while and look to have been abandoned by their original creators. Please accept this PR so that github actions can handle dealing with stale PRs and issues.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by DennisSSDev** [comment]
The cron schedule defined in the yaml is that it'll run every day at midnight

File: .github/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: "Stale issue handler"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
```
Comment (marioizquierdo) [inline .github/stale.yaml:7]: Add a comment with info and a link to the documentation for this GitHub action works.

File: .github/stale.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        stale-pr-message: 'This PR is stale because it has been open for 30 days with no activity. Remove stale label / comment or this will be closed in 5 days'
+        days-before-stale: 30
```
Comment (marioizquierdo) [inline .github/stale.yaml:16]: 30 days may be too aggressive for Twirp, what about 60? (we can change this later btw)

File: .github/stale.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        days-before-close: 5
+        stale-pr-label: 'no-pr-activity'
+        stale-issue-label: 'no-issue-activity'
```
Comment (marioizquierdo) [inline .github/stale.yaml:19]: add a newline at the end (recommended to configure your editor to do it automatically on save)

File: .github/stale.yaml (hunk 2 above)
Comment (DennisSSDev) [reply .github/stale.yaml:16]: 60 sounds good

**Review by marioizquierdo — APPROVED** [review]
💯  :shipit: 💯

**Pull request merged**